content/pages/download-va-letters.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Download VA Letters
+layout: page-breadcrumbs.html
+---
+<script src="https://standards.usa.gov/assets/js/vendor/uswds.min.js" type="text/javascript"></script>
+
+<div class="main">
+<div class="section">
+<div class="row" markdown="1">
+
+# Download Your VA Letters
```

Comment:
We should be able to remove this if we update the metadata as mentioned above.
Also, should the title be "Download VA Letters" or "Download Your VA letters"? If the latter, should the links in the nav and footer also be updated to match?

Reply:
@bethpotts ^